src/Simulation/Simulators.Tests/Circuits/Arrays.qs

```diff
@@ -0,0 +1,77 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Quantum.Simulation.Simulators.Tests.Circuits {
+    open Microsoft.Quantum.Diagnostics;
+    open Microsoft.Quantum.Intrinsic;
+
+    operation MapF<'T, 'U>(mapper : ('T -> 'U), source : 'T[]) : 'U[] {
+        mutable result = new 'U[Length(source)];
+        for (i in 0 .. Length(source) - 1) {
+            let m = mapper(source[i]);
+            set result = result w/ i <- m;
+        }
+        
+        return result;
+    }
+
+    operation LengthTest() : Unit {
```

Comment:
Is `LengthTest` used anywhere?

Reply:
Oh, nvm. I see it was originally in the removed file and has been updated to use the new syntax here. Cool!